README.md

```diff
@@ -301,7 +301,7 @@ certain domains at startup that depend on concepts.  We have a separate process
 concepts are loaded in.
 
 ```properties
-initializer.domains=!programs,programworkflows,programworkflowstates,drugs
```

Comment:
Assumedly the other change here to replace this with the full disable is in another PR?